src/lib/services/charge-versions.js

```diff
@@ -65,11 +66,14 @@ const getByLicenceRef = async licenceRef =>
  *
  * @param {String} licenceId The licence id
  */
-const getByLicenceId = async licenceId => service.findMany(
-  licenceId,
-  chargeVersionRepo.findByLicenceId,
-  chargeVersionMapper
-);
+const getByLicenceId = async licenceId => {
```

Comment:
I think it would be good if we could add an `optional include notes flag` with default to false, to only return line 75 if set to true otherwise return without the notes. We do not always want to get the note i.e. for billing or on the view licence page you don't need to get the notes so might slow it down in those areas especially for billing.